### ISSUE TITLE
Enhance text on bank transfer order complete

### ISSUE THREAD

**Issue by philip-n** [inbaz/ers#142]
The following plain text is shown after an order by bank transfer was successful. The according bold parts should be links to the according pages (and maybe vary depending on if the user is already logged in or not?).

---

You can request this information again

All you need to do is:

**Setup a password** for your account with your email address niggl@albapasser.de.
**Login** to your account.
View your **order detail**.
View your **payment detail**.

**Comment by inbaz** [comment]
@smadamit What do you think about not to have the login possibility for all purchasers here? I don't know if all the needed profile stuff is getting ready until Kickoff. The purchaser can reach his/her order with the link in the email to restart the credit card payment I think that's enough.

**Comment by smadamit** [comment]
@inbaz Yes our decision is: implementation of all profile features -> milestone:SecondLife

@philip-n For me the issue can be closed?

**Comment by smadamit** [comment]
let's have a look to this issue again after implementation of the profile

**Comment by smadamit** [comment]
see #57
